services/pull/pull.go

```diff
@@ -64,7 +64,7 @@ func NewPullRequest(ctx context.Context, opts *NewPullRequestOptions) error {
 	}
 
 	// user should be a collaborator or a member of the organization for base repo
-	if !issue.Poster.IsAdmin {
+	if (!issue.Poster.IsAdmin) && pr.Flow == issues_model.PullRequestFlowGithub {
```

Comment:
Maybe you should only change for line 82?

Reply:
> Maybe you should only change for line 82?

why？ looks the result is same.